app/src/main/java/camp/nextstep/edu/calculator/CalculatorActivity.kt

```diff
@@ -11,6 +11,11 @@ class CalculatorActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         binding = ActivityMainBinding.inflate(layoutInflater)
         setContentView(binding.root)
+        setUpButton()
+
+    }
+
+    private fun setUpButton() {
```

Comment:
```suggestion
    private fun setupListener() {
```
함수명만 봤을 때에는 "버튼을 셋업한다"의 의미이지만, 실제 내부 구현부는 클릭리스너를 등록하고 있어요.
그렇다면 "리스너를 셋업한다"의 의미가 더 적합하지 않을까요?

Reply:
아하~ 구체적으로 함수명을 지어야겠네요 감사하빈다~